src/validation.ts

```diff
@@ -118,24 +120,9 @@ export function checkRecipientAmountsAndDuplicates(
 }
 
 function checkRecipientAmount(recipient: Recipient, index: number) {
-  const {
-    welcomeTravelerRewards,
-    earlyUserRewards,
-    liquidityProviderRewards,
-    communityRewards,
-    referralRewards,
-    opRewards,
-    arbRewards,
-  } = recipient.metadata.amountBreakdown;
-
-  const amountBreakdownSum = BigNumber.from(welcomeTravelerRewards)
-    .add(earlyUserRewards)
-    .add(liquidityProviderRewards)
-    .add(communityRewards)
-    .add(referralRewards)
-    .add(opRewards)
-    .add(arbRewards);
-
+  const amountBreakdownSum = Object.values(
+    recipient.metadata.amountBreakdown
+  ).reduce((amountAcc, reward) => amountAcc.add(reward), BigNumber.from(0));
```

Comment:
I think this is also no longer necessary

Reply:
Removed here [51eae69](https://github.com/across-protocol/merkle-distributor/pull/15/commits/51eae69dea24965dbd14383fe0b8a28a869174d8)